kpatch.py

```diff
@@ -93,6 +93,21 @@ def configure(self):
             demands.available_repos = False
 
 
+    def _list_missing_kpp_pkgs(self):
+        kpps = []
+
+        installed_kernels = self.base.sack.query().installed().filter(name=KERNEL_PKG_NAME)
+
+        for kernel_pkg in installed_kernels:
+            kpp_pkg_name = _kpp_name_from_kernel_pkg(kernel_pkg)
+            kpp_pkgs_query = self.base.sack.query().filter(name=kpp_pkg_name, arch=kernel_pkg.arch).latest().difference(self.base.sack.query().installed())
```

Comment:
This also lists the currently installed patch. I am not sure what is the issue. Does anyone see there something obviously wrong?

Reply:
@sm00th or @joe-lawrence do you maybe happen to know?

Reply:
Not really sure. Does the set in difference call include it? I also think filtering the 'installed' set on name=kpp_pkg_name is a good idea so that difference() has less to chew through.

Reply:
> Does the set in difference call include it? 

Checked and it does. It feels like `difference()` doesn't do what it should?  I'll try to create stand alone reproducer and contact dnf developers directly.

> I also think filtering the 'installed' set on name=kpp_pkg_name is a good idea so that difference() has less to chew through.

Makes sense, I'll do that.